app/src/main/res/values/booleans.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <bool name="isTablet">false</bool>
+  <bool name="shouldSplit">false</bool>
```

Comment:
``shouldSplitInteractions`` might be clearer. Otherwise this is pretty vague for being in a centralized location like this. Also, isn't this more like "shouldCheckForSplittingInteraction" since the actual decision to do so depends on other factors (e.g. interaction ID & physical device size)?

Reply:
`shouldCheckForSplittingInteraction` is better in my opinion.

Reply:
Done, renamed to `shouldCheckForSplittingInteraction`

Reply:
Resolving this on behalf of Ben.